src/Sarif/ExtensionMethods.cs

```diff
@@ -16,6 +16,8 @@ namespace Microsoft.CodeAnalysis.Sarif
 {
     public static class ExtensionMethods
     {
+        private static readonly Uri BaseUri = new Uri("https://github.com/microsoft/sarif-sdk/");
+
```

Comment:
use [https://example.com](https://example.com) here (and provide a comment). actually! how about declaring this as an inlined string const in your helper?